crates/reporting/src/error/parse.rs

```diff
@@ -718,6 +705,33 @@ fn to_expr_report<'a>(
                 severity,
             }
         }
+        EExpr::PizzaDbgWithArg(pos) => {
+            let surroundings = Region::new(start, *pos);
+            let region = to_keyword_region(lines.convert_pos(*pos), "dbg");
+
+            let doc = alloc.stack([
+                alloc.concat([
+                    alloc.reflow("The "),
+                    alloc.keyword("dbg"),
+                    alloc.reflow(" keyword expects one expression, but it was given two instead:"),
+                ]),
+                alloc.region_with_subregion(lines.convert_region(surroundings), region, severity),
+                alloc.concat([
+                    alloc.reflow("You can use "),
+                    alloc.keyword("dbg"),
+                    alloc.reflow(" to inspect the value at different stages of a pipeline by doing something like "),
+                    alloc.parser_suggestion("foo |> dbg |> bar"),
+                    alloc.reflow("."),
+                ]),
+            ]);
```

Comment:
I would drop the "doing something" part, the usual pattern with this kind of tips in errors is "You can X, like...".

Reply:
I would also change the example to something more realistic, like:

```roc
products
|> List.map .price
|> dbg
|> List.sum
|> dbg
```

Reply:
I've removed this error message